src/components/Table/index.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+
+export const Table = styled.table`
+  border: none;
+  position: relative;
+  width: 90%;
+  background: #ffffff;
+`;
+
+export const TableRow = styled.tr`
+  height: 22px;
+  top: 58px;
+  background: #ffffff;
+  cursor: pointer;
+  &:hover {
+    background-color: #f3f3f3;
+  }
+`;
+export const RowData = styled.td`
+  padding-left: 5px
+  max-width: 100%;
+  white-space: nowrap;
+  height: 20px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 18px;
+  letter-spacing: 0.01em;
+  color: #000000;
+`;
+
```

Comment:
This is great! But I think it'd be good to also add the other table base tags 😉

Reply:
Thanks, Davide. With the latest commit, I have added the "td" tag as well.